internal/provider/resource_consul_cluster.go

```diff
@@ -129,6 +128,12 @@ func resourceConsulCluster() *schema.Resource {
 					return strings.ToLower(old) == strings.ToLower(new)
 				},
 			},
+			"auto_hvn_to_hvn_peering": {
+				Description: "Enables automatic HVN to HVN peering when creating a secondary cluster in a federation.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
I am honestly not sure what to do with this. There are two different things that can happen:

* false -> true: recreating the cluster will setup the peering
* true -> false: the already existing peering will be left untouched and nothing changes.

Reply:
I'm also a little wary of using ForceNew here. In particular for the `true -> false`, it would be nice if we didn't force the user to recreate the clusters just because they've decided to do their own networking setup. Might be worth pulling in someone from product/design to discuss the possible user flows 🤔